getting_started/step_by_step/7_room_two.rst

```diff
@@ -49,150 +49,177 @@ concert or traffic lights.
 --------------------------------
 If you played the game now, you'd find it was a bit broken. The game would
 start in the pub room, but you'd have no way to leave it to get to the park.
-If you were able to get to the park and tried to get back, the game wouldn't
-know where in the pub to place your character. Lets fix this by creating
-a doorway in the pub and make it so that using it takes you to the park, and
-clicking on the edge of the park brings you back to the pub's doorway.
+If you were somehow able to get to the park and tried to get back, the game
+wouldn't know where in the pub to place your character.
+
+Lets fix this by creating a doorway in the pub and make it so that using it
+takes you to the park, and clicking on the edge of the park brings you back to
+the pub's doorway.
 
 The process here will be the same process we followed in the park.
 
 Load your pub.tscn scene in the Godot editor.
 
 Create an ESCexit as a child of the ESCRoom. Give it a Global ID of
-"pub_door".
-
-Create a new ESC file and call it "pub_exit.esc" In pub_exit.esc, add a
-new ``:use`` event to our ESC file and use the  ``change_scene`` command
-to make the game switch from this pub room to the park.
-
-.. code-block::
-
-    :use
-    change_scene "res:///rooms/park.tscn"
+"pub_door". Set its ``Target scene`` to park.tscn. This will send the player to
+the park when they click the door.
 
-Set the player locations in the pub
------------------------------------
-When you created the pub room you set a ESClocation for the character's
-starting point. If you placed it, for example, at the bottom of the
-stairs, while this is fine when starting a new game, if the player
-comes back into the pub through the door, you don't want them to
-appear at the bottom of the stairs. We therefore need a location at
-the doorway that we can send them to.
-
-Create a new ESClocation as a child of the pub's ESCRoom. Set its
-``Global ID`` to pub_doorway. This will be where the user appears
-when they come into the pub from the park.
-
-Create a CollisionPolygon2D as a child of the ESClocation. Draw the
+Create a CollisionPolygon2D as a child of the ESCExit. Draw the
 polygon around the doorway. This defines where you can click to
 activate the exit to leave the pub and go to the park.
 
 .. image:: img/create_room_pub_door.png
    :alt: The polygon you click to leave the room.
 
-Create an ESClocation as a child of the ESCExit and place its marker
-just under the doorway. This will be where the character walks to
+When you created the pub room you set a ESClocation for the character's
+starting point. If you placed it, for example, at the bottom of the
+stairs, while this is fine as a location to put the character when starting a
+new game. If the player comes back into the pub from the park through, they
+should be at the doorway rather than appear at the bottom of the stairs. We
+therefore need an ``ESCLocation`` at the doorway that we can send them to.
```

Comment:
```suggestion
starting point (e.g. at the bottom of the stairs). If the player comes back into the pub from the park 
scene, they should be at the doorway rather than appear at the bottom of the stairs. We
therefore need an ``ESCLocation`` at the doorway that we can send them to.
```